data/mods/ssb/moves.ts

```diff
@@ -98,6 +98,39 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
 		type: "Rock",
 	},
 
+	// n10siT
+	"unbind": {
+		accuracy: 100,
+		basePower: 60,
+		category: "Special"
+		desc: "",
+		shortDesc: "",
+		name: "Unbind",
+		pp: 15,
+		priority: 0,
+		flags: {protect: 1},
+		onTryMove() {
+			this.attrLastMove('[still]');
+		},
+		onPrepareHit(target, source) {
+			this.add('-anim', source, 'Hyperspace Hole', source);
+			this.add('-anim', source, 'Hyperspace Fury', source);
+		},
+		onHit(target, pokemon, move) {
+			if (pokemon.baseSpecies.baseSpecies === 'Hoopa') {
+				pokemon.formeChange(pokemon.species.otherformes[0], this.effect, false, '[msg]');
+				this.boost({spe: 1}, pokemon, pokemon, move);
+			}
+		},
+		onModifyType(move, pokemon) {
+		if (pokemon.baseSpecies.baseSpecies !== 'Hoopa') return;
+			move.type = pokmeon.species.types[1];
```

Comment:
if the user got soaked, it would crash, use pokmeon.species.types.length. Also in base form it would be psychic, and dark in unbound form. This code would make the move ghost in base.